deck/models.py

```diff
@@ -39,18 +39,20 @@ class Deck(models.Model):
     piles = JSONField(null=True, blank=True) 
     deck_contents = JSONField(null=True, blank=True) #All the cards that should be included when shuffling and whatnot
     shuffled = models.BooleanField(default=False)
+    include_jokers = models.BooleanField(null=False, default=False)
```

Comment:
`null=False` by default correct? Is it generally a best practice to be explicit about this?

Reply:
You're right. I'll change this.